heartbeat/Makefile

```diff
@@ -1,7 +1,7 @@
 BEAT_NAME=heartbeat
 BEAT_TITLE=Heartbeat
 SYSTEM_TESTS=true
-TEST_ENVIRONMENT=false
+TEST_ENVIRONMENT=true
```

Comment:
To make sure the test environment is disable on OS X builds you need to add a `?` before `=`. See https://github.com/elastic/beats/blob/master/metricbeat/Makefile#L5